src/components/styles.scss

```diff
@@ -0,0 +1,96 @@
+.board {
+  display: flex;
+  justify-content: space-around;
+  min-height: 550px;
+  padding: 10px;
+  border-radius: 8px;
+  min-width: 100%;
+  box-sizing: border-box;
+  background-color: #ccd6c9;
+  .board_title {
+    display: flex;
+    justify-content: center;
+    font-size: 20px;
+    color: #2c3e5d;
+  }
+}
+
+.list_item {
+  min-width: 320px;
+  min-height: 500px;
+  padding: 10px 15px 10px 15px;
+  border-radius: 4px;
+  background-color: #ebecf0;
+  margin: 0 5px;
+  & h1 {
+    color: #2c3e5d;
```

Comment:
useless &.
Use either "h1" witohut "&"
or better assign the className

Reply:
fixed